taxcalc/functions.py

```diff
@@ -959,11 +1002,6 @@ def AMTI(c60000, _exact, e60290, _posagi, e07300, x60260, c24517,
 
     _amt25pc = min(_alminc, c62740) - min(_alminc, c62720)
 
-    if c62730 == 0:
```

Comment:
`c62730` corresponds to Line 38, Form 625. And an in-line instruction on the form reads that 
"If line 38 is zero or blank, skip lines 59 through 61 and go to line 62. Otherwise, go to line 59", where 59 through 62 involves the calculation of _amt25 stuff. So I think we should reverse the changes here.